### DISCUSSION TITLE
Welcome to bbox Discussions!

### DISCUSSION THREAD

**Discussion by superisaac** [haobtc/bbox#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rayston92** [comment]
hey, what does bbox stand for, sort of a new chain or just antoher commuity-driven project?

**Reply by superisaac** [reply]
It's a nano service framework

**Reply by rayston92** [reply]
got it, a multi-lang micro service toolkit.
